.github/workflows/main.yml

```diff
@@ -0,0 +1,18 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "14.x"
```

Comment:
Bundlephobia's core package retains support for Node >= 10.x

Reply:
It just a version that runs the specs. I think since we bundle the code anyway, it not important which version do we have here. I just chose v14 because it's LTS and should be more stable and run the specs faster.